ports/net/macathena-athrun/Portfile

```diff
@@ -1,17 +1,44 @@
-# $Id$
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 PortSystem		1.0
+PortGroup		github 1.0
+PortGroup		python 1.0
 
 name			macathena-athrun
-version			20060725
-distname		athrun-${version}
+github.setup           macathena athrun 3c9c5b72414454df3c4faad6d92929c5b6e0dc2b
+version			10.3.2.1
```

Comment:
Ooh, I think this is where we get to use epoch! 10 < 20060725, so to cause an upgrade to happen, you need to add `epoch 1`.